packages/nc-gui-v2/assets/img/temp/251.json

```diff
@@ -0,0 +1 @@
+[{"city":"Kabul"}]
```

Comment:
Dummy file?

Reply:
have to clean up the assets and public folder